core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -845,7 +847,7 @@ public synchronized void save() {
      * Loads the data from the disk into this object.
      */
     public synchronized void load() throws IOException {
-        UpdateSite defaultSite = new UpdateSite(ID_DEFAULT, config.getUpdateCenterUrl() + "update-center.json");
+        UpdateSite defaultSite = new UpdateSite(LEGACY_ID_DEFAULT, config.getUpdateCenterUrl() + "update-center.json");
         XmlFile file = getConfigFile();
```

Comment:
Not sure, I would argue that is confusing.
We do not need to create a default update center in the subclass, cause that is only used when no other update center is available. So if e.g. CJEUpdateSite fails to start then the default update center is added (pointing to oss).